3.Arrays-and-Loops/filter.js

```diff
@@ -0,0 +1,3 @@
+let array = [1,6,7,8,3,4,5,6]
+let arr = array.filter(value => value <=3 );//filtering elements whose value is more then 3

```

Comment:
Please read task, you need elements whose value more than 3, your result is [1, 3]